employees/common/strings.py

```diff
@@ -33,3 +34,12 @@ class ReportDetailStrings(Enum):
     DELETE_POPUP_TITLE = ugettext_lazy("Delete report")
     DELETE_POPUP_YES = ugettext_lazy("Yes")
     DELETE_POPUP_NO = ugettext_lazy("No")
+
+
+class TaskActivitiesStrings(Enum):
```

Comment:
As discussed, it this can ever change I would make it a table, and even create a separate entries from some defaults per each project, so if someone want to change it in single project, it won't have impact on others.